sandbox/settings.py

```diff
@@ -598,6 +598,27 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         environ_prefix=None,
     )
 
+    # Course run offer values that would be shown on course detail page
+    RICHIE_DEFAULT_COURSE_RUN_OFFER = values.Value(
+        "free",
+        environ_name="RICHIE_DEFAULT_COURSE_RUN_OFFER",
+        environ_prefix=None,
+    )
+
+    # Course run price currency value that would be shown on course detail page
+    RICHIE_DEFAULT_COURSE_RUN_PRICE_CURRENCY = values.Value(
+        "EUR",
+        environ_name="RICHIE_DEFAULT_COURSE_RUN_PRICE_CURRENCY",
+        environ_prefix=None,
+    )
+
+    # Course run price value that would be shown on course detail page
+    RICHIE_DEFAULT_COURSE_RUN_PRICE = values.Value(
+        0.0,
+        environ_name="RICHIE_DEFAULT_COURSE_RUN_PRICE",
+        environ_prefix=None,
+    )
```

Comment:
`RICHIE_DEFAULT_COURSE_RUN_OFFER` and `RICHIE_DEFAULT_COURSE_RUN_PRICE` look not used. Furthermore, I wonder if this is really useful to be able to override to default values through a settings. WDYT?

Reply:
I agree that the `RICHIE_DEFAULT_COURSE_RUN_PRICE` shouldn't be on a setting.
@JoaoGarcao the `RICHIE_DEFAULT_COURSE_RUN_OFFER` should be included only on the next PR that you are working.

Reply:
Change done.

Reply:
I'm going to readd the `RICHIE_DEFAULT_COURSE_RUN_OFFER`, but use it on the course model.